changelog/unreleased/pull-3612

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add GCS option for encryption keys
+
+These changes allow users to provide 256-bit AES crypto 
+keys to Google Cloud Storage to enable Customer-supplied 
+encryption.
+
+https://github.com/restic/restic/pull/3612

```

Comment:
Please add a reference to the issue here.

Reply:
Added.